install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+function install {
+  printf "\e[92m[~] Name your project: \e[0m\n"
+
+  read name
+
+  printf "\e[33m[~] Downloading awesomeness...\e[0m\n"
+
+  git clone https://github.com/mostjs/package-starter $name \
+  && cd $name \
+  && rm -rf ./.git \
+  && git init \
+  && npm install \
+  && sed -i -- "s/mostPackage/${name}/g" package.json 
```

Comment:
Should we flip these last two?  i.e. do `sed` first, since it's very unlikely to fail, and `npm install` probably fails more often.  If we flip them, and `npm install` fails, the `sed` still will have succeeded, leaving the user's dir in a more easily recovered state: the user just has to run `npm install` again manually.